spec/features/users_can_view_fund_level_activities_spec.rb

```diff
@@ -17,6 +18,12 @@
       page_displays_an_activity(activity_presenter: ActivityPresenter.new(fund_activity))
     end
 
+    it "does not show a link to download as XML" do
+      visit organisation_activity_path(fund_activity.organisation, fund_activity)
+
+      expect(page).to_not have_content t("default.button.download_as_xml")
+    end
```

Comment:
Follow-up to activity summary comment - this would only check if it appears on the financials tab, right? Whereas if it was in the summary, it wouldn't matter which tab it is

Reply:
It's true, and if we had more time to ask for design opinions I'd like to revise this!